src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -26,25 +26,113 @@ public ScorerPipelineStep(Var<IDataView> data, Var<ITransformModel> model)
         public Var<ITransformModel> Model { get; }
     }
 
+
+    /// <summary>
+    /// <see cref="LearningPipeline"/> class is used to define the steps needed to perform a desired machine learning task.<para/>
```

Comment:
><see cref="LearningPipeline"/> class [](start = 8, length = 36)

I'd really prefer "This class", but if you really want the actual name of the class referenced, for grammar reasons maybe prefix with "The" so that it's The LearningPipeline class" vs. "LearningPipeline class."